.github/workflows/.test.yml

```diff
@@ -8,19 +8,21 @@ jobs:
   check-style:
     runs-on: ubuntu-latest
     container:
-      image: golang:1.20-bookworm
+      image: golang:1.21-bookworm
     steps:
     - name: ci/checkout-repo
       uses: actions/checkout@v4
     - name: ci/setup-go-and-cache
       uses: ./.github/actions/go-setup
+    - name: Install goimports
+      run: go install golang.org/x/tools/cmd/goimports@latest
```

Comment:
Can you help me understand this part? I see that there is also a Makefile target for this.

Reply:
For some reason make command didn't work on the github action context so I forced to use this.  These changes in github actions come from a purpose to unify as much as possible the GAs across our repositories to be more maintainable in the long run. As far as I remember this step was required for next steps to work (check-style). Don't remember though why it failed without this.

Reply:
I removed `goimports` as go fmt is enough